### PR TITLE
Added ranges for alpha and lambda in docs

### PR DESCRIPTION
**Pull request by edjeordjian**

### CODE REVIEW

File: doc/parameter.rst [hunk 1]
```diff
@@ -121,10 +121,12 @@ Parameters for Tree Booster
 * ``lambda`` [default=1, alias: ``reg_lambda``]
 
   - L2 regularization term on weights. Increasing this value will make model more conservative.
+  - range: [0,∞]
```
Comment (trivialfis) [inline doc/parameter.rst:124]: Thank you for the improvement, could you please consider using a LaTeX symbol with `:math:`?

Comment (trivialfis) [reply doc/parameter.rst]: it's `:math:` ,not `math:`.

Comment (edjeordjian) [reply doc/parameter.rst]: Thank you for your patience 😅

**Review by trivialfis — APPROVED** [review]
Thank you for the improvement!

**Pull request merged**